src/pybind/mgr/volumes/fs/operations/group.py

```diff
@@ -18,10 +18,13 @@ class Group(GroupTemplate):
     # Reserved subvolume group name which we use in paths for subvolumes
     # that are not assigned to a group (i.e. created with group=None)
     NO_GROUP_NAME = "_nogroup"
+    INTERNAL_DIRS = ["_index", "_deleting", "_legacy"]
```

Comment:
I'm not sure if we need this  class variable? Also hard coding "index", "_deleting", "_legacy" doesn't seem right

Reply:
> I'm not sure if we need this class variable? Also hard coding "index", "_deleting", "_legacy" doesn't seem right

+1